run_scans.py

```diff
@@ -106,6 +107,12 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, scanmin, scanmax, nevts,
     cmd.append( "--scanmax=%d"%(scanmax) )
     cmd.append( "--nevts=%d"%(nevts) )
     cmd.append( "--throttle=%i"%(throttle) )
+    if stepSize > 0:
+      step = stepSize
+      if (step + scanmin > scanmax):
+        step = scanmax - scanmin
+      cmd.append( "--stepSize=%d"%(step) )
+      pass
```

Comment:
These lines in run_scans.py are unnecessary. You already coded this check in ultraLatency.py.

You should only be passing the parameter here